### DISCUSSION TITLE
Can I use the eye mesh model with v23.5.19?

### DISCUSSION THREAD

**Discussion by RaXorX** [Bing-su/adetailer#240] [General]
I'm using the Photoshop plugin https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin
by @AbdullahAlfaraj and since the later versions of adetailer do not work as of now with it, the currently working version is v23.5.19. But that doesn't has the eyemesh model. Is it possible to use the eyemesh model from the newer version with the older version of adetailer?

**Comment by Bing-su** [comment]
아니요

**Comment by RaXorX** [comment]
Yes I reckoned after reading the source, it's using mediapipe to detect the meshes which was added later on.
